gqueries/modules/multi_year_charts/flexible_capacity_chart/myc_flexible_capacity_in_energy_heat_flexibility_p2h_heatpump.gql

```diff
@@ -0,0 +1,8 @@
+# The flexible capacity in energy heat flexibility p2h heatpump electricity
+
+- unit = MW
+- query =
+    SUM(
+        Q(energy_heat_flexibility_p2h_heatpump_ht_electricity_capacity),
```

Comment:
Note that I would assume that we use the **electricity input capacity** in this chart. These queries show the total input capacity:
```
- query = V(energy_heat_flexibility_p2h_heatpump_ht_electricity, "number_of_units * input_capacity")
- unit = MW
```
Because the electricity input conversion is ~ 22% for this heat pump, it does make a significant difference whether to use the input capacity or the electricity input capacity.